package escrow;

message AvailableTradePartnersRequest {
}

message AvailableTradePartnersResponse {
	repeated bytes client_id = 1;
}